modules/reliquary/components/DeligateClearButton.tsx

```diff
@@ -0,0 +1,47 @@
+import { HStack, Text } from '@chakra-ui/react';
+import React, { useEffect } from 'react';
+import { BeetsSubmitTransactionButton } from '~/components/button/BeetsSubmitTransactionButton';
+import { ToastType, useToast } from '~/components/toast/BeetsToast';
+import { useDelegateClear } from '../lib/useDelegateClear';
+import { useDelegation } from '../lib/useDelegation';
+
+interface Props {
+    size?: string;
+    rounded?: string;
+    w?: string;
+}
+
+export default function DeligateClearButton({ ...rest }: Props) {
+    const { clearDelegate, ...clearDelegateQuery } = useDelegateClear();
+    const { refetch } = useDelegation();
```

Comment:
Question: Not possible to bundle the delegation hooks into one single hook?

Reply:
I'll add a TODO